tests/integration/large_message_azure_test.py

```diff
@@ -56,6 +56,27 @@ def azure_bucket_name():
     yield bucket_name
     container_client.delete_container()
 
+def test_azure_serializer_value_mixed_case_topic(wait_for_api, azure_bucket_name):
+    my_mixed_case_topic = "myMixedCaseTopic-1"
+    serializer = LargeMessageSerializer(my_mixed_case_topic, config)
+    binary_input = b"This is a test for s3"
```

Comment:
s3 is confusing here. Just use a generic text without any reference to azure or s3